rest/main/main.go

```diff
@@ -103,11 +111,15 @@ func main() {
 
 	// Prepare HTTPS server
 	restServer := &http.Server{
-		Addr:      address,
-		Handler:   router,
-		TLSConfig: &tlsConfig,
+		Addr:        address,
+		Handler:     router,
+		TLSConfig:   &tlsConfig,
+		ReadTimeout: readTimeout,
+		ErrorLog:    serverLog,
 	}
 
+	glog.Infof("Read timeout = %v", readTimeout)
+	glog.Infof("Authentication modes = %v", clientAuth)
```

Comment:
Is this really needed? Wouldn't it be better if those messages were shown only when debugging is on?

Reply:
These are only printed once during service startup. It was very useful for me to identify misconfigurations. But I can change them as verbose level logs.

Reply:
That would be perfect. Thx!